docs/content/doc/setup/reverse-proxies.md

```diff
@@ -80,6 +80,22 @@ server {
 <b>NOTE:</b> If you change the max upload size in Vikunja's settings, you'll need to also change the <code>client_max_body_size</code> in the nginx proxy config.
 </div>
 
+## NGINX Proxy Manager (NPM)
+
+1. Create a Proxy Host within NPM and point it to whatever URL you plan to use. 
+2. Verify that the page will pull up in your browser. (Do not bother trying to log in. It won't work. Trust me.)
+3. Identify the container name for your NPM installation. e.g. `NGINX-PM`
+4. From the command line, enter sudo docker exec -it NGINX-PM /bin/bash and navigate to the proxy hosts folder where the `.conf` files are stashed. Probably `/data/nginx/proxy_host`.
+5. Locate the `.conf` file where the server_name matches your Vikunja Proxy Host. Once found, add the following code, unchanged, just above the existing location block in that file. (They are listed by number, not name.)  
+```
+location ~* ^/(api|dav|\.well-known)/ {
+        proxy_pass http://api:3456;
+        client_max_body_size 20M;
+    }
+```
+6. After saving the edited file, return to NPM's browser window and refresh the page to verify your Proxy Host for Vikunja is still online. 
+7. The last step is to adjust the address in the top of the login subscreen to match your proxy address.
```

Comment:
Actually, this should not be required if the proxy is set up correctly.

Reply:
I would agree, but propose that it is the `app.ini` file or original Vikunja container that needs to be set up with the correct TLD. It's looking for the proxied host address and I hadn't set that up initially. (I've changed my Docker Stack so that it now has the correct address.)